app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivity.kt

```diff
@@ -114,15 +115,13 @@ class AdministratorControlsActivity :
     /** Returns an [Intent] to start this activity. */
     fun createAdministratorControlsActivityIntent(context: Context, profileId: Int?): Intent {
       val intent = Intent(context, AdministratorControlsActivity::class.java)
-      intent.putExtra(NAVIGATION_PROFILE_ID_ARGUMENT_KEY, profileId)
       intent.decorateWithScreenName(ADMINISTRATOR_CONTROLS_ACTIVITY)
+      intent.decorateWithUserProfileId(
```

Comment:
Out of curiosity, if all activities have been updated to use this new decorator is it possible for us just to pass ProfileId around instead of the internal profile int? It might actually simplify this PR a bit since it keeps the data flow a lot cleaner (and avoids awkward cases like this where we need to construct a ProfileId).

Reply:
Thanks Ben this makes sense, updating refactor to this pattern.